frontend/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "frontend",
```

Comment:
Change the name to something like "Switchboard" or "Switchboard App" or "Switchboard Frontend"

Reply:
Fixed in abe57fb